frontend/ui/widget/filechooser.lua

```diff
@@ -559,14 +559,20 @@ end
 
 -- Used in ReaderStatus:onOpenNextDocumentInFolder().
 function FileChooser:getNextFile(curr_file)
+    local show_finished = self.show_finished
```

Comment:
I wouldn't mind making it explicit that this is a class member by *always* accessing it as `FileChooser.` (ditto for `show_unsupported`,  e.g., in `FileChooser:show_file`)